test/src/dom/query.spec.ts

```diff
@@ -0,0 +1,53 @@
+/*-----------------------------------------------------------------------------
+| Copyright (c) 2014-2015, PhosphorJS Contributors
+|
+| Distributed under the terms of the BSD 3-Clause License.
+|
+| The full license is in the file LICENSE, distributed with this software.
+|----------------------------------------------------------------------------*/
+'use strict';
+
+import expect = require('expect.js');
+
+import {
+  hitTest
+} from '../../../lib/dom/query';
+
+
+describe('dom/query', () => {
+
+  describe('hitTest()', () => {
+
+    it('should return `true` when point is inside the node', () => {
+      let div = document.createElement('div');
+      div.className = 'hit-test';
+      document.body.appendChild(div);
+      expect(hitTest(div, 50, 50)).to.be(true);
+      document.body.removeChild(div);
+    });
+
+    it('should return `false` when point is outside the node', () => {
+      let div = document.createElement('div');
+      div.className = 'hit-test';
+      document.body.appendChild(div);
+      expect(hitTest(div, 150, 150)).to.be(false);
+      document.body.removeChild(div);
+    });
+
+    it('should use closed intervals for left and top only', () => {
+      let div = document.createElement('div');
+      div.className = 'hit-test';
+      document.body.appendChild(div);
+      expect(hitTest(div, 0, 0)).to.be(true);
+      expect(hitTest(div, 100, 0)).to.be(false);
+      expect(hitTest(div, 99, 0)).to.be(true);
+      expect(hitTest(div, 0, 100)).to.be(false);
+      expect(hitTest(div, 0, 99)).to.be(true);
+      expect(hitTest(div, 100, 100)).to.be(false);
+      expect(hitTest(div, 99, 99)).to.be(true);
+      document.body.removeChild(div);
+    });
+
+  });
+
+});
```

Comment:
can you add the test for `scrollIfNeeded()`, which was just added to this module?